Justfile

```diff
@@ -1,4 +1,4 @@
-build input="details.example.yml" output="output.pdf":
+build input="details.yml" output="output.pdf":
```

Comment:
If you haven't seen it, try the `render-travel-expenses` script, it can render `yml` files from anywhere in your filesystem, it's quite convenient :)